src/opensynth/models/faraday/gaussian_mixture/__init__.py

```diff
@@ -1,9 +1,2 @@
 # Copyright Contributors to the Opensynth-energy Project.
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
I think we could add some of the modules here to make it more convenient for end users to import I think!

Reply:
Done.